<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsStartup</class>
 <widget class="QWidget" name="SettingsStartup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="StartupBehavior">
     <property name="title">
      <string>Startup Behavior</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="StartupBehavior_None">
        <property name="text">
         <string>Don't open any worlds</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="StartupBehavior_Reopen">
        <property name="text">
         <string>Reopen last worlds</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="StartupBehavior_List">
        <property name="text">
         <string>Open worlds in list:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="OpenAtStartup">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SelectionMode::ExtendedSelection</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="OpenAtStartup_buttons" native="true">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="OpenAtStartup_add">
           <property name="text">
            <string>Add</string>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::ListAdd"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="OpenAtStartup_remove">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Remove</string>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::ListRemove"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="OpenAtStartup_up">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Move Up</string>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::GoUp"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="OpenAtStartup_down">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Move down</string>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::GoDown"/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>StartupBehavior_List</sender>
   <signal>toggled(bool)</signal>
   <receiver>OpenAtStartup</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>StartupBehavior_List</sender>
   <signal>toggled(bool)</signal>
   <receiver>OpenAtStartup_buttons</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
